### ISSUE TITLE
check if openglsw32.dll is definetely needed. IF yes, update readme's, windeploy_helper and build env

### ISSUE THREAD

**Issue by medusadigital** [monero-project/monero-core#927]
check if openglsw32.dll is definetely needed. Currently it is added by hand for each release. 

If it is needed, readme, windows deployment script and buildenv will need to be updated.

**Comment by pazos** [comment]
it was used in Helium Hydra, not in Lithium Luna yet.
This is a [library](https://www.mesa3d.org/llvmpipe.html) that supports opengl without needing a GPU.

This is really important for people trying to use the GUI via remote desktop or with VMware/Virtualbox

**Comment by selsta** [comment]
Yes, opengl32sw.dll is required. I'll see if I can improve the DEPLOY instructions.
